models/repo/repo_list.go

```diff
@@ -695,7 +695,9 @@ func GetUserRepositories(opts *SearchRepoOptions) (RepositoryList, int64, error)
 	}
 
 	cond := builder.NewCond()
-	cond = cond.And(builder.Eq{"owner_id": opts.Actor.ID})
+	if opts.Actor != nil {
+		cond = cond.And(builder.Eq{"owner_id": opts.Actor.ID})
+	}
```

Comment:
I am not sure whether we should do this.

This function is named as `GetUserRepositories: returns a list of repositories of given user.`

If this `cond` is removed, then it will list all user's repositories.

Reply:
if actor == nil we have an other issue!!!